charts/smartmetserver/templates/ingress.yaml

```diff
@@ -1,23 +1,24 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
- name: smartmetserver-ingress
- namespace: smartmet
- annotations:
-   cert-manager.io/cluster-issuer: letsencrypt
+  name: {{ .Values.smartmetserver.name }}-{{ .Values.ingress.name }}
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
 spec:
- tls:
- - hosts:
-   - data.lab.rauhalat.org
-   secretName: smartmetserver-ingress-tls
- rules:
- - host: data.lab.rauhalat.org
-   http:
-     paths:
-     - backend:
-         service:
-           name: smartmetserver
-           port:
-             number: 8080
-       path: /
-       pathType: Prefix
+  ingressClassName: nginx
```

Comment:
ingressClassName should be optional parameter and configurable (other charts in this repo have "ingressClassName: nginx" but will be refactored! Default functionality should be that no ingressClassName is set in the ingress.

Reply:
ingressClassName changed to be optionally configurable. See commit: https://github.com/fmidev/helm-charts/commit/f7e35bc58ddc26d22063350022f1079273991431